config/postcss.config.js

```diff
@@ -0,0 +1,7 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2016--, QIIME development team.
+//
+// Distributed under the terms of the Modified BSD License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+// ----------------------------------------------------------------------------
```

Comment:
Delete this file, unless I'm mistaken we don't use postcss yet (we might someday).